File: iniciante/prob_1066.py
even = 0
positives = 0
negatives = 0
for i in range(5):
    x = int(input())
    if x % 2 == 0:
        even += 1
        odd = 5 - even
    if x > 0:
        positives += 1
    if x < 0:
        negatives += 1
print(
    f"{even} valor(es) par(es)\n{odd} valor(es) impar(es)\n{positives} valor(es) positivo(s)\n{negatives} valor(es) negativo(s)"
)

File: java/prob_1066.java
import java.util.Scanner;

public class prob_1066 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        float valor;
        int pares, impares, positivos, negativos;
        pares = 0;
        impares = 0;
        positivos = 0;
        negativos = 0;

        for (int i = 0; i < 5; i++){
            valor = sc.nextFloat();

            if (valor % 2 == 0)
                pares++;
            if (valor % 2 != 0)
                impares++;
            if (valor > 0)
                positivos++;
            if (valor < 0)
                negativos++;
        }

        System.out.println(pares + " valor(es) par(es)");
        System.out.println(impares + " valor(es) impar(es)");
        System.out.println(positivos + " valor(es) positivo(s)");
        System.out.println(negativos + " valor(es) negativo(s)");

        sc.close();
    }
}
